raiden/raiden_service.py

```diff
@@ -534,7 +535,18 @@ def stop(self) -> None:
 
         log.debug("Raiden Service stopped", node=to_checksum_address(self.address))
 
-    def add_notification(self, notification: Notification) -> None:
+    def add_notification(
+        self,
+        notification: Notification,
+        log_opts: Optional[Dict] = None,
+        click_opts: Optional[Dict] = None,
+    ) -> None:
+        log_opts = log_opts or {}
+        click_opts = click_opts or {}
+
+        log.info(notification.summary, **log_opts)
+        click.secho(notification.body, **click_opts)
```

Comment:
For a first implementation this is fine.

However ideally we should come up with a way to combine logging and "printed" (`click.(s)echo`) mesages. 
Since this way in the default configuration those messages will be printed twice.

Also the log level / color should depend on the severity of the notification.
It would be nice to re-log notifications the user hasn't acknowledged yet periodically.
But this needs more design / planning beforehand.